error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/util/MoreASTHelpers.java

```diff
@@ -0,0 +1,38 @@
+package tech.picnic.errorprone.bugpatterns.util;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+
+import com.google.common.collect.ImmutableList;
+import com.google.errorprone.VisitorState;
+import com.sun.source.tree.ClassTree;
+import com.sun.source.tree.MethodTree;
+import javax.lang.model.element.Name;
+
+/** A set of helper methods for working with the AST. */
+public final class MoreASTHelpers {
+  private MoreASTHelpers() {}
+
+  /**
+   * Finds methods with the given name in the given class.
+   *
+   * @param enclosingClass The class to search in.
+   * @param methodName The method name to search for.
+   * @return The method trees of the methods with the given name in the class.
```

Comment:
```suggestion
   * @return The {@link MethodTree}s of the methods with the given name found in the class.
```
Maybe even "in the enclosing class"? Or "given class".